2019-05-22 19:10:57.003 [error] <0.345.0> gen_server <0.345.0> terminated with reason: {socket_error,timeout}
2019-05-22 19:10:57.003 [error] <0.344.0> gen_server <0.344.0> terminated with reason: {socket_error,timeout}
2019-05-22 19:10:57.003 [error] <0.329.0> gen_server <0.329.0> terminated with reason: {socket_error,timeout}
2019-05-22 19:10:57.004 [error] <0.326.0> gen_server <0.326.0> terminated with reason: {socket_error,timeout}
2019-05-22 19:10:57.940 [error] <0.345.0> CRASH REPORT Process <0.345.0> with 0 neighbours exited with reason: {socket_error,timeout} in gen_server:handle_common_reply/8 line 751
2019-05-22 19:10:57.940 [error] <0.344.0> CRASH REPORT Process <0.344.0> with 0 neighbours exited with reason: {socket_error,timeout} in gen_server:handle_common_reply/8 line 751
2019-05-22 19:10:57.941 [error] <0.316.0> Supervisor {<0.316.0>,amqp_connection_type_sup} had child main_reader started with amqp_main_reader:start_link(#Port<0.11>, <0.326.0>, <0.340.0>, {method,rabbit_framing_amqp_0_9_1}, <<"client 127.0.0.1:51790 -> 127.0.0.1:5672">>) at <0.345.0> exit with reason {socket_error,timeout} in context child_terminated
2019-05-22 19:10:57.941 [error] <0.328.0> Supervisor {<0.328.0>,amqp_connection_type_sup} had child main_reader started with amqp_main_reader:start_link(#Port<0.12>, <0.329.0>, <0.341.0>, {method,rabbit_framing_amqp_0_9_1}, <<"client 127.0.0.1:51791 -> 127.0.0.1:5672">>) at <0.344.0> exit with reason {socket_error,timeout} in context child_terminated
2019-05-22 19:10:57.941 [error] <0.328.0> Supervisor {<0.328.0>,amqp_connection_type_sup} had child main_reader started with amqp_main_reader:start_link(#Port<0.12>, <0.329.0>, <0.341.0>, {method,rabbit_framing_amqp_0_9_1}, <<"client 127.0.0.1:51791 -> 127.0.0.1:5672">>) at <0.344.0> exit with reason reached_max_restart_intensity in context shutdown
2019-05-22 19:10:57.941 [error] <0.316.0> Supervisor {<0.316.0>,amqp_connection_type_sup} had child main_reader started with amqp_main_reader:start_link(#Port<0.11>, <0.326.0>, <0.340.0>, {method,rabbit_framing_amqp_0_9_1}, <<"client 127.0.0.1:51790 -> 127.0.0.1:5672">>) at <0.345.0> exit with reason reached_max_restart_intensity in context shutdown
2019-05-22 19:10:57.947 [error] <0.326.0> CRASH REPORT Process <0.326.0> with 0 neighbours exited with reason: {socket_error,timeout} in gen_server:handle_common_reply/8 line 751
2019-05-22 19:10:57.948 [error] <0.313.0> Supervisor {<0.313.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.316.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,60000,none,[fun amqp_a..>,...],...}) at <0.326.0> exit with reason {socket_error,timeout} in context child_terminated
2019-05-22 19:10:57.948 [error] <0.313.0> Supervisor {<0.313.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.316.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,60000,none,[fun amqp_a..>,...],...}) at <0.326.0> exit with reason reached_max_restart_intensity in context shutdown
2019-05-22 19:10:57.948 [error] <0.329.0> CRASH REPORT Process <0.329.0> with 0 neighbours exited with reason: {socket_error,timeout} in gen_server:handle_common_reply/8 line 751
2019-05-22 19:10:57.948 [error] <0.327.0> Supervisor {<0.327.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.328.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,60000,none,[fun amqp_a..>,...],...}) at <0.329.0> exit with reason {socket_error,timeout} in context child_terminated
2019-05-22 19:10:57.948 [error] <0.327.0> Supervisor {<0.327.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.328.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,10,60000,none,[fun amqp_a..>,...],...}) at <0.329.0> exit with reason reached_max_restart_intensity in context shutdown
